imx.mk

```diff
@@ -37,8 +37,8 @@ BOOT_IMG		?= $(ROOT)/out/boot.img
 ################################################################################
 # Targets
 ################################################################################
-all: tfa u-boot linux optee-os buildroot flash-image
-clean: tfa-clean buildroot-clean u-boot-clean optee-os-clean
+all: optee-os tfa u-boot linux buildroot flash-image
+clean: ddr-firmware-clean optee-os-clean tfa-clean u-boot-clean buildroot-clean
```

Comment:
That's OK, please keep this line only and rename the commit "imx.mk" add ddr-firmware-clean to clean target". With that it is `Reviewed-by: Jerome Forissier <jerome.forissier@linaro.org>`.

Reply:
done